src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +71,55 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block{
```

Comment:
Can you think of a benefit for creating separate files for the classes VS nested classes (what you are doing now)?